Data/Sentinel1_processing_guide.qmd

```diff
@@ -0,0 +1,63 @@
+
+## Copernicus Sentinel-1 Mosaics
+
+Sentinel-1 GRD monthly and yearly mosaics (available soon).
+
+It is planned that mosaics will be processed with the following Sentinel Hub processing options:
+
+- Orthorectification with Copernicus 10m/30m DEM
+- Radiometric Terrain Correction
+- No speckle filtering applied 
+
+## [Sentinel Hub processing options](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options){target="_blank"} 
+
+Sentinel Hub offers the following processing options in the [Sentinel-1 GRD processing chain](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-chain){target="_blank"}:
+
+- [Backscatter coefficients](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options){target="_blank"}:
+
+    - beta0
+    - sigma0
+    - gamma0 (ellipsoid)
+    - gamma0 (terrain) &rarr; **gamma0 RTC can only be performed if orthorectification is enabled**
+
+- [Lee Speckle Filtering](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options){target="_blank"} applied on source data after calibration and noise removal
+- [Orthorectification](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options){target="_blank"} with Range-Doppler terrain correction using one of the following DEMs:
+
+    - Copernicus 10m/30m DEM (10m resolution inside [39 European states](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model){target="_blank"} including islands and 30m elsewhere.)
+    - Copernicus 30m DEM
+    - Copernicus 90m DEM 
+
+## [CloudFerro processing options](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"} 
+
+CloudFerro offers CARD-BS and COH6/COH12 products as a [service on demand](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"}: 
+
+- [Sentinel-1 (CARD-BS) BackScatter](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-card-bs){target="_blank"}
+
+    - This processing option contains gamma0 geometric terrain correction (orthorectification), which is identical to the gamma0 (ellipsoid) backscatter coefficient with orthorectification option in Sentinel Hub processing options. No Radiometric Terrain Correction (RTC) is applied to this product. 
+    - [Additional information](https://creodias.eu/eodata/sentinel-1/sentinel-1-l2-backscatter-bs/){target="_blank"}.
+
+- [Sentinel-1 (CARD-COH) Coherence](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html){target="_blank"}
+
+    - The Sentinel-1 CARD COH (Copernicus Analysis Ready Data Coherence) processor generates a Sentinel-1 Level 2 product describing the coherence of a pair of images - 12 days apart. Concurrently, a terrain-correction (using DEM) is performed. 
```

Comment:
@alek-cesarz Can you provide information on the DEM that is used for the processing?

Reply:
same ad for CARD-BS : COPDEM-30 (Copernicus DEM GLO-30)